.gitignore

```diff
@@ -15,3 +15,15 @@ bin/
 
 /text-ui-test/ACTUAL.txt
 text-ui-test/EXPECTED-UNIX.TXT
+data/data.bin
+src/main/java/duke/.idea/.gitignore
+src/main/java/duke/.idea/misc.xml
+src/main/java/duke/.idea/modules.xml
+src/main/java/duke/.idea/vcs.xml
+src/main/java/duke/.idea/codeStyles/codeStyleConfig.xml
+src/main/java/duke/data/data.bin
+text-ui-test/data/data.bin
+src/test/.idea/.gitignore
+src/test/.idea/misc.xml
+src/test/.idea/modules.xml
+src/test/.idea/vcs.xml
```

Comment:
This is a minor thing but you can shorten these by just including the directory, e.g. `data/`, `src/test/.idea/` and the files under the directory will be ignored automatically